lib/characteristic.dart

```diff
@@ -11,16 +11,35 @@ abstract class _CharacteristicMetadata {
   static const String value = "value";
 }
 
+/// A characteristic of a local peripheral's [Service].
+///
+/// It contains a single value and any number of [Descriptor]s describing that
```

Comment:
Descriptors describe the Characteristic itself, not its value, I think. They're kind of metadata, I guess?

Reply:
To be honest I found it in CoreBluetooth documentation. I assumed that Apple is not wrong. But maybe... 🤔